Workbooks/Virtual Machines/Key Metrics/Key Metrics.workbook

```diff
@@ -0,0 +1,394 @@
+{
+  "version": "Notebook/1.0",
+  "isLocked": false,
+  "items": [
+    {
+      "type": 1,
+      "content": {
+        "json": "# Virtual Machines Metrics"
+      },
+      "conditionalVisibility": null
+    },
+    {
+      "type": 9,
+      "content": {
+        "version": "KqlParameterItem/1.0",
+        "query": "",
+        "crossComponentResources": [],
+        "parameters": [
+          {
+            "id": "85bc1eb3-c168-4054-a925-2bcc86074223",
+            "version": "KqlParameterItem/1.0",
+            "name": "VirtualMachines",
+            "type": 5,
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "value": [
+              "value::all"
+            ],
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "resourceTypeFilter": {
+                "microsoft.compute/virtualmachines": true
+              },
+              "additionalResourceOptions": [
+                "value::all",
+                "value::3"
+              ]
+            }
+          },
+          {
+            "id": "0d13757a-59d3-4fdc-ae21-1de067448751",
+            "version": "KqlParameterItem/1.0",
+            "name": "TimeRange",
+            "type": 4,
+            "isRequired": true,
+            "isHiddenWhenLocked": false,
+            "typeSettings": {
+              "selectableValues": [
+                {
+                  "durationMs": 300000,
+                  "createdTime": "2018-08-21T23:40:02.726Z",
```

Comment:
do all the time ranges actually work?  some metrics don't allow really long ranges like 90 days or whatever.  some do?